src/pages/GitHubPage.ts

```diff
@@ -1,28 +1,27 @@
 import { expect, test, type Locator, type Page } from '@playwright/test';
-import { AbstractPage } from './BasePage';
+import { BasePage } from './BasePage';
  
-export class GitHubPage extends AbstractPage {
-  readonly page: Page;
+export class GitHubPage extends BasePage {
+  readonly url: string = '/github';
```

Comment:
You also can create separate `constants.ts` file with all the routes defined in one place:

```
export const ROUTES = {
  cart: '/cart',
  github: '/github',
  menu: '/menu',
}
```

Then in the file class this will look like:

`url = ROUTES.github;`

Reply:
fixed